### DISCUSSION TITLE
Duplicate Users in App Internal App Table

### DISCUSSION THREAD

**Discussion by josh99smith** [Budibase/budibase#11356] [Help] [answered]
I have created an automation that was uppose to update update some fields in the internal Users table.

However, my automation failed and instead of updating users, it created brand new ones.

These new ones have "Unkown role" as the role and the user email address as the email field. These are the only two fields it populated.

These are active users on my app, so I dont want to delete the correct user row, but I want to get rid of these duplicate rows.

Is there anyway to get rid of these without deleting the original user row? 

This is how they are showing up in my table..
![image](https://github.com/Budibase/budibase/assets/4806827/497f3bd6-4865-4e10-ba30-cfe7edf6a9f9)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @josh99smith,

Could you take some screenshots of the automation you used that caused this? (If you are able to replicate this could you create an issue and fill out the template so we can get it fixed.)
What happens when you publish your app?

You may be able to delete the row using the public API https://docs.budibase.com/reference/delete_tables-tableid-rows-rowid, right click the row and copy the _id of the duplicated row. Then using this along with the other details required and see if you can delete that row through the public api.

I hope this helps.

**Comment by josh99smith** [comment]
Thank you for your response! The automation ideal worked. I was able to delete those users with no problem by just creating an automation that targeted those IDs for deletion! Much appreciated. 

I would be more than happy to to try an recreate this issue. I am not sure what the template is that you are refering to is though.

Below is also a screen grab of the automation that caused this. It was the update rows action.

![Budibase](https://github.com/Budibase/budibase/assets/4806827/8dc146b7-a92a-4a95-9f4c-9b53dacf8330)

Thank you again for your help, this community and support team is amazing!

**Reply by ConorWebb96** [reply]
No worries, glad to have helped!

When i mentioned the template I was just talking about our issues that can be created here; https://github.com/Budibase/budibase/issues/new/choose. 

If you can't recreate don't worry. I'll see if I can recreate this on my end using the screenshot you provided of your automation. 

Thanks for the screenshots!
